src/Summoner/License.hs

```diff
@@ -1,51 +1,81 @@
+{-# LANGUAGE LambdaCase #-}
+
 module Summoner.License
-       ( License (..)
-       , customizeLicense
-       , licenseNames
-       , githubLicenseQueryNames
-       ) where
+    ( LicenseName(..)
+    , License(..)
+    , cabalLicenseName
+    , customizeLicense
+    , githubLicenseQueryNames
+    , parseLicense
+    )
+where
```

Comment:
Export list should be formatted according to style-guide:

* https://github.com/serokell/serokell-util/blob/master/serokell-style.md#export-lists